plugins/GSdx/Window/GSwxDialog.cpp

```diff
@@ -0,0 +1,917 @@
+/*
+ *	Copyright (C) 2020 Pcsx2 Team
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with GNU Make; see the file COPYING.  If not, write to
+ *  the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA USA.
+ *  http://www.gnu.org/copyleft/gpl.html
+ *
+ */
+
+#include "GSwxDialog.h"
+
+void add_tooltip(wxWindow *widget, int tooltip)
```

Comment:
I think it would be good to make these helper functions static or use an anonymous namespace so it is unique to the translation unit

Reply:
I moved them to an anonymous namespace. Since my header has lots of "Type *name, *name2, *name3, *name4; ", trying to put the * on the Type starts looking fairly inconsistent. I just ran the files through clang-format, though, so they should adhere to any standards set in the format file now.